frontend/src/__mocks__/mockPodLogs.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable camelcase */
+
+type MockLogConfigType = {
+  namespace: string;
+  podName: string;
+  containerName: string;
+};
+export const mockPodLogs = ({
+  namespace = 'test-project',
+  podName = 'conditional-execution-pipeline-0858f-flip-coin-pod',
```

Comment:
podName here should be generic, such as `test-podName`, or something similar. You can modify the name when you call this function.